src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -56,6 +88,76 @@ namespace Utils
         {
         }
 
+        /**
+         * @brief Class destructor. Frees column family handles.
+         *
+         */
+        ~RocksDBWrapper()
+        {
+            std::for_each(m_columnsHandles.begin(),
+                          m_columnsHandles.end(),
+                          [this](rocksdb::ColumnFamilyHandle* handle)
+                          {
+                              // Default column should not be freed.
```

Comment:
Note: Not freeing the default column results in 32 bytes leaked each time, and freeing it causes sporadic segfaults. Anyway, the docs clearly state we [shouldn't free it](https://github.com/facebook/rocksdb/blob/88bc91f3cc2b492b8a45ba2c49650f527df97ad8/include/rocksdb/db.h#L407).